core/templates/pages/topic-editor-page/subtopic-editor/subtopic-preview-tab.directive.html

```diff
@@ -0,0 +1,57 @@
+<h3>
+  Back to <a ng-click="navigateToSubtopic()"><[editableTitle]></a>
+</h3>
+<div>
+  <div class="content-box">
+    <div class="left-menu">
+      <div class="left-menu-item" ng-click="changeContent(SHOW_THUMBNAIL)"><span>Card</span></div>
+      <div class="left-menu-item" ng-click="changeContent(SHOW_CONTENT)"><span>Content</span></div>
+    </div>
+    <div class="content">
+      <preview-thumbnail ng-if="showThumbnail" name="editableName"
+                         description="editableDescription"
+                         preview-footer="topic.getCanonicalStoryIds().length"
+                         filename = "editableThumbnailFilename"
+                         thumbnail-bg-color="editableThumbnailBgColor"
+                         bg-color="'#2F6687'">
+      </preview-thumbnail>
+
+      <angular-html-bind ng-if="!showThumbnail" html-data="pageContents.getHtml()">
+      </angular-html-bind>
+    </div>
+  </div>
+</div>
+
+<style>
```

Comment:
Have all the styles be encapsulated in the directive tag, since some of these seem like common names.